integration-tests/tests/src/hooks/open-realm-before.ts

```diff
@@ -83,3 +84,34 @@ export function openRealmBefore(config: OpenRealmConfiguration = {}): void {
   before(openRealmBefore.name, openRealmHook(config));
   after("closeRealmAfter", closeThisRealm);
 }
+
+export async function setupRealmHook(this: AppContext & MultiRealmContext): Promise<void> {
+  this.openedInfo = [];
+
+  this.getRealm = async (config: OpenRealmConfiguration): Promise<Realm> => {
+    const user = await this.getUser(Realm.Credentials.anonymous(false));
+    const realmAndConfig = await openRealm(config, user);
+    this.openedInfo.push(realmAndConfig);
+    return realmAndConfig.realm;
+  };
+
+  this.closeAllRealms = () => {
+    // this.openedInfo?.forEach(({ realm, config }) => {
+    //   if (!realm?.isClosed) {
+    //     realm.close();
+    //   }
+    //   Realm.deleteFile(config);
+    // });
```

Comment:
There is another thread in this PR that is not visible here because it's on an outdated version of this file, regarding if here using `clearTestState` is enough.